components/ColorModeSwitch.vue

```diff
@@ -2,7 +2,7 @@
   <ColorScheme>
     <Icon
       v-if="isMobile"
-      class="cursor-pointer"
+      class="cursor-pointer size-6"
```

Comment:
could we have a default size setting for Icon component? Like `size: small | medium | large`

Reply:
Sure 👍🏻 I was also thinking about wrapping that component

Reply:
updated, added `<KIcon />` component